ntbs-service/Models/Notification.cs

```diff
@@ -1,18 +1,127 @@
-﻿using System.Collections.Generic;
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using ntbs_service.Models.Enums;
 
 namespace ntbs_service.Models
 {
     public class Notification
     {
         public int NotificationId { get; set; }
-
         public virtual PatientDetails PatientDetails { get; set; }
         public virtual ClinicalDetails ClinicalDetails { get; set; }
-
         public virtual List<NotificationSite> NotificationSites { get; set; }
         public virtual Episode Episode { get; set; }
         public virtual PatientTBHistory PatientTBHistory { get; set; }
         public virtual ContactTracing ContactTracing { get; set; }
         public virtual SocialRiskFactors SocialRiskFactors { get; set; }
+
+        public string FullName {
+            get {
+                return String.Join(", ", new string[] {PatientDetails.FamilyName.ToUpper(), PatientDetails.GivenName});
+            }
+        }
+
+        public string FormattedNhsNumber {
+            get {
+                return CreateFormattedNhsNumberString();
+            }
+        }
+
+        public string FormattedPostcode {
+            get {
+                return CreatePostcodeString();
+            }
+        }
+
+        public string SitesOfDiseaseList { 
+            get {
+                return CreateSitesOfDiseaseString();
+            } 
+        }
+
+        public string DrugRiskFactorTimePeriods { 
+            get {
```

Comment:
Wouldn't it make more sense for the methods that are defined on the owned models to be defined there, rather than all in here? Might we not want to make use of some of these  when we only have e.g. Clinical Details or PatientDetails available, so we don't have to go via notification? It would also make this model quite a bit shorter, and just add an extra ClinicalDetails into Notification.ClinicalDetails.DaysFromOnsetToTreament.
@jasiekmiko , @ChristianMcc , what do you think? Not sure whether you already had a chat in person

Reply:
Yep we've had a chat and I think it's an ongoing decision, I'll have another chat with Jan now to see if the current way is still preferred or to switch to the way you've described

Reply:
Yeah, my gut feeling is that we'll not be interacting with the individual tables much at all outside of the edit pages, and instead be manipulating the whole Notification, which means the current version is nicer, as the user does't have to care how the property is split up. Agreed that it's one to keep an eye on in the future and potentially shift over to individual pages if the need arises!
(Also, the `=>` syntax for getter-only properties should help with the lenght of the model, at least visually!)

Reply:
Ok, sounds good to me :)